db/migrate/20190313134953_downgrade_cover_quality_for_older_playlists.rb

```diff
@@ -0,0 +1,6 @@
+class DowngradeCoverQualityForOlderPlaylists < ActiveRecord::Migration[5.2]
+  def change
+    Playlist.where(pic_id: 0..69807).update(cover_quality: :ancient)
+    Playlist.where(pic_id: 69806..72848).update(cover_quality: :legacy)
+  end
+end
```

Comment:
Somehow this didn't take, so a bunch of covers went to broken images.

```irb(main):003:0> me.id
=> 6689
irb(main):004:0> me.cover_quality
=> "modern"```

```
irb(main):002:0> Playlist.where(pic_id: 0..69807).count                          
=> 0
```

Reply:
Ah, you already fixed it. Sorry about this.